README.fr.md

```diff
@@ -219,7 +219,6 @@ Tout client supportant une API de type Fever ; Sélection :
 * [bcrypt.js](https://github.com/dcodeIO/bcrypt.js)
 * [phpQuery](https://github.com/phpquery/phpquery)
 * [Services_JSON](https://pear.php.net/pepr/pepr-proposal-show.php?id=198)
-* [password_compat](https://github.com/ircmaxell/password_compat)
```

Comment:
I don't think `git rm lib/password_compat.php` is part of this PR yet?

Reply:
My bad, GH hides big changes so at a glance it looks like there are none.

Reply:
Yes, `password_compat` is removed because it is not needed from PHP 5.5+